site/app/templates/officeHoursQueue/QueueFooter.twig

```diff
@@ -311,5 +313,37 @@
       });
   }
 
+  function updateMessage(remove) {
+      location.reload();
```

Comment:
What if instead of reloading the page, we make a network request to quickly fetch the queue message and have javascript render it in. This is how the office hour queue works. It helps reduce server load and provide a smoother experience to user as it's less sudden reloads. Just make sure that the code you use to set the content sets the text and not actual HTML to prevent an XSS issue.